systemtest/src/test/resources/upgrade/BundleDowngrade.yaml

```diff
@@ -30,18 +30,18 @@
 #    kafkaKRaftAfter: path to Kafka and KafkaNodePool resources (KRaft mode), collected in one file, in the version of Strimzi, to which we are doing the downgrade
 # --- Structure ---
 - fromVersion: HEAD
-  toVersion: 0.44.0
+  toVersion: 0.45.0
   fromExamples: HEAD
-  toExamples: strimzi-0.44.0
+  toExamples: strimzi-0.45.0
   fromUrl: HEAD
-  toUrl: https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.44.0/strimzi-0.44.0.zip
+  toUrl: https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.45.0/strimzi-0.45.0.zip
   fromKafkaVersionsUrl: HEAD
   additionalTopics: 2
   imagesAfterOperations:
-    zookeeper: strimzi/kafka:0.44.0-kafka-3.8.0
-    kafka: strimzi/kafka:0.44.0-kafka-3.8.0
-    topicOperator: strimzi/operator:0.44.0
-    userOperator: strimzi/operator:0.44.0
+    zookeeper: strimzi/kafka:0.45.0-kafka-3.8.0
+    kafka: strimzi/kafka:0.45.0-kafka-3.8.0
```

Comment:
we have 3.9.0 now, should this be updated? or it's about upgrading strimzi but taking the same Kafka version?

Reply:
it's about changing the Strimzi version, the Kafka version is changed when new Kafka version is added.